### ISSUE TITLE
Error from server (Forbidden): 

### ISSUE THREAD

**Issue by everythingeverywhere** [vmware-tanzu/pinniped#479]
After watching a `kubectl get all` we got this unexpected output:
```bsh
Every 1.0s: kubectl get all                                              eduk8s-labs-w01-s001-5c64cd54f6-8222r: Fri Mar  5 22:12:56 2021
NAME                                READY   STATUS    RESTARTS   AGE
pod/k8s-demo-app-84c5bf4c7b-g64tr   1/1     Running   0          27s
NAME                   TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)   AGE
service/k8s-demo-app   ClusterIP   10.107.45.82   <none>        80/TCP    27s
NAME                           READY   UP-TO-DATE   AVAILABLE   AGE
deployment.apps/k8s-demo-app   1/1     1            1           27s
NAME                                      DESIRED   CURRENT   READY   AGE
replicaset.apps/k8s-demo-app-84c5bf4c7b   1         1         1       27s
Error from server (Forbidden): webhookauthenticators.authentication.concierge.pinniped.dev is forbidden: User "system:serviceaccount:edu
k8s-labs-w01:eduk8s-labs-w01-s001" cannot list resource "webhookauthenticators" in API group "authentication.concierge.pinniped.dev" in
the namespace "eduk8s-labs-w01-s001"
```

Getting these errors in an eduk8s lab, it was suggested that  CRs should be removed from "all" alias to prevent this.

**Comment by GrahamDumpleton** [comment]
To make very clear what the problem is, it is that when a user is not a cluster admin, but has restrictive RBAC roles, they will get these sorts of errors when performing any operations against "all" alias.

In general it seems to be bad practice to add custom resources types to the "all" alias, doubly so when they are resource types which are only of interest to cluster admins. The only things that should be in "all" alias are builtin core Kubernetes resources.

**Comment by paulczar** [comment]
While not solving the problem this will give all authenticated users read access to the resource and remove the errors.  I'm not 100% if this will also solve it for tools like OPA/Gatekeeper that need pretty wide open read-only access to resources, or if the group is for real users only.

```
kubectl create clusterrole fix-pinniped-role --verb=get,list,watch \
  --resource=webhookauthenticators.authentication.concierge.pinniped.dev

kubectl create clusterrolebinding fix-pinniped-role-binding \
  --clusterrole=fix-pinniped-role \
  --group=system:authenticated
```

**Comment by GrahamDumpleton** [comment]
This may already have been addressed in newer pinniped version than was being used. I actually partly recollect raising this specific type of issue via a back channel in a VMware project some time back, so may have been pinniped.

**Comment by mattmoyer** [comment]
@everythingeverywhere can you share the version of Pinniped you're using? As @enj said, we believe this was fixed in v0.3.0.

**Comment by mattmoyer** [comment]
Closing for now, please feel free to re-open if you have any new details or questions.
